src/components/Pagination/index.js

```diff
@@ -1,20 +1,87 @@
-import { Wrapper } from "./styled";
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+import usePagination, { DOTS } from "../../hooks/usePagination";
```

Comment:
With the setup you have for your `hooks` folder, I think you should be able to do:
```javascript
import { usePagination } from '../../hooks';
```
Let me know if doable.